<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Starfiles</class>
 <widget class="QMainWindow" name="Starfiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>421</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>722</width>
    <height>421</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>722</width>
    <height>421</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Starfiles</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets/resources.qrc">
    <normaloff>:/images/images/logo/favicon.ico</normaloff>:/images/images/logo/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Starfiles, #TabHome, #TabFiles,
#TabSettings {background-color:#fff;}

QTabWidget::pane {
  	border: 1px solid rgb(160, 160, 160);
  	top:-1px;
	bottom:-1px;
	right:-1px;
	padding-top:1px;
  	background: rgb(245, 245, 245);
}

QTabBar::tab {
	background: rgb(245, 245, 245);
  	border: 1px solid lightgray;
	border-bottom:1px solid rgb(160, 160, 160);
  	padding: 17px;
}

QTabBar::tab:selected {
	background: rgb(230, 230, 230);
}

QTableWidget &gt; * {
	font: 9pt &quot;Titillium Web&quot;;
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Titillium Web</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="TabHome">
       <attribute name="title">
        <string> Home</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="BtnUpload">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Titillium Web</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Upload File</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0" colspan="4">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="BtnCancelUpload">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Titillium Web</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Cancel !</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Titillium Web</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;file sharing done simple.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Titillium Web SemiBold</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Starfiles&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabFiles">
       <attribute name="title">
        <string>Files</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableWidget" name="FileList">
          <property name="font">
           <font>
            <family>Titillium Web</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>File ID</string>
           </property>
           <property name="background">
            <color>
             <red>245</red>
             <green>245</green>
             <blue>245</blue>
            </color>
           </property>
          </column>
          <column>
           <property name="text">
            <string>File Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Download Link</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabSettings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>611</width>
          <height>261</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Titillium Web</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Application Settings :</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="font">
            <font>
             <family>Titillium Web</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Private Key :</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLineEdit" name="TextPk">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Private key goes here</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="BtnCancelSettings">
             <property name="font">
              <font>
               <family>Titillium Web</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnSaveSettings">
             <property name="font">
              <font>
               <family>Titillium Web</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="font">
            <font>
             <family>Titillium Web</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Database :</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QCheckBox" name="ChkSave2db">
              <property name="font">
               <font>
                <family>Titillium Web</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Save to local database</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="BtnDeleteDatabase">
              <property name="font">
               <font>
                <family>Titillium Web</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>delete my database</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>403</y>
          <width>130</width>
          <height>13</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Titillium Web</family>
         </font>
        </property>
        <property name="text">
         <string>Starfiles App, Version : 1.0.0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>521</x>
          <y>401</y>
          <width>149</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Titillium Web</family>
         </font>
        </property>
        <property name="text">
         <string>Copyright © 2021 Anas Yousef</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
